patches/components-sync_bookmarks-bookmark_change_processor.cc.patch

```diff
@@ -0,0 +1,86 @@
+diff --git a/components/sync_bookmarks/bookmark_change_processor.cc b/components/sync_bookmarks/bookmark_change_processor.cc
+index acb064faac3ee975ec07be4977fc0970309dfb15..81744cb05711149192d93d0a2e688cf706720248 100644
+--- a/components/sync_bookmarks/bookmark_change_processor.cc
++++ b/components/sync_bookmarks/bookmark_change_processor.cc
+@@ -87,6 +87,7 @@ void BookmarkChangeProcessor::UpdateSyncNodeProperties(
+   bookmark_specifics.set_creation_time_us(src->date_added().ToInternalValue());
+   dst->SetBookmarkSpecifics(bookmark_specifics);
+   SetSyncNodeFavicon(src, model, dst);
++  brave_sync::AddBraveMetaInfo(src, model, false);
+   SetSyncNodeMetaInfo(src, dst);
+ }
+ 
+@@ -244,6 +245,7 @@ void BookmarkChangeProcessor::CreateOrUpdateSyncNode(const BookmarkNode* node) {
+     // Acquire a scoped write lock via a transaction.
+     syncer::WriteTransaction trans(FROM_HERE, share_handle(), &new_version);
+     sync_id = model_associator_->GetSyncIdFromChromeId(node->id());
++    ScopedPauseObserver pause(bookmark_model_, this);
+     if (sync_id != syncer::kInvalidId) {
+       UpdateSyncNode(
+           node, bookmark_model_, &trans, model_associator_, error_handler());
+@@ -383,7 +385,8 @@ int64_t BookmarkChangeProcessor::UpdateSyncNode(
+ 
+ void BookmarkChangeProcessor::BookmarkMetaInfoChanged(
+     BookmarkModel* model, const BookmarkNode* node) {
+-  BookmarkNodeChanged(model, node);
++  // TODO(darkdh): make sure the meta info changed is what we want to sync
++  // BookmarkNodeChanged(model, node);
```

Comment:
this seems very dangerous to me, is there another way to handle this?

Reply:
we don't sync any chromium meta info in current brave sync.
https://github.com/brave/brave-core/blob/a0275cc96254a625add41e49740183af3d943a03/components/brave_sync/client/bookmark_change_processor.cc#L382